### DISCUSSION TITLE
Increase the batch limit

### DISCUSSION THREAD

**Discussion by ySergi** [lllyasviel/Fooocus#2180] [Ideas]
Why is it capped at 32 images? It should be unlimited, at the user's choice. I would like to generate 200 images at a time, not every 32 images click Generate again.

**Comment by poor7** [comment]
I think it's possible to use the "Generate forever" mode. Enabled and disabled by right-clicking on the Generate button.

**Reply by ySergi** [reply]
Hmm well, it's an alternative... Thanks.

Also, we should be able to set a custom generations limit.

**Reply by poor7** [reply]
https://github.com/lllyasviel/Fooocus/discussions/1537

**Comment by SGThirkell** [comment]
You can add a line to config.txt like
"default_max_image_number": 40,

This should work

**Reply by mashb1t** [reply]
@016R35H this. See https://github.com/lllyasviel/Fooocus/pull/1616
